lib/agoneum/account/user.ex

```diff
@@ -12,17 +18,43 @@ defmodule Agoneum.Account.User do
     field :password, :string, virtual: true
     field :password_hash, :string
 
+    field :active, :boolean, default: false
```

Comment:
This is a good discussion point. Without this flag, how would we mark people as in/out for games? Do we just want another flag?